date.js

```diff
@@ -1,2 +1,4 @@
 restaurants = getAllRestaurants();
 categories = getAllCategories();
+
+inizializeRequest('aa', {}, 'GET', function() {});
```

Comment:
1. file name : `data.js` instead of date.js
2. let's prepare this for real token, not a hardcoded string :)
3. why is it here? :D